### PR TITLE
flameshot: add examples and fix more info link

### PR DESCRIPTION
**Pull request by mmahmoudian**
Hi,

I'm one of the maintainers of Flameshot and by accident saw some info are wrong and some are outdated. I fixed them as far as I could.

About the checkboxes below, I was not sure about one of the (unchecked) and the documentation didn't help either. I hope I have done it by the book though. let me know what should be fixed and I'll comply.

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

P.s: this project rocks. Thanks for keeping it rolling.

### CODE REVIEW

**Comment by CleanMachine1** [comment]
Hi, thanks for the contribution!

I will give some of my suggestions now however before merging can happen, you need to sign the CLA.



ps, I really like flameshot, ever since starting with Linux, I found it and its the much better replacement for snipping tool.

File: pages/linux/flameshot.md [hunk 1]
```diff
@@ -2,24 +2,32 @@
 
 > Screenshot utility with a GUI.
 > Supports basic image editing, such as text, shapes, colors, and imgur.
-> More information: <https://flameshot.js.org>.
+> More information: <https://flameshot.org>.
 
-- Launch flameshot in GUI mode:
+- Launch flameshot launcher window:
```
Comment (CleanMachine1) [inline pages/linux/flameshot.md:7]:
```suggestion
- Launch flameshot with an interactive mode:
```

Although technical terms have their place, the description is best when the user can gather some information from it.

File: pages/linux/flameshot.md [hunk 2]
```diff
@@ -2,24 +2,32 @@
...
 
 `flameshot launcher`
 
-- Take a screenshot by clicking and dragging:
+- The GUI mode to take a screenshot by clicking and dragging and annotating interactively:
```
Comment (CleanMachine1) [inline pages/linux/flameshot.md:11]:
```suggestion
- Launch flameshot and immediately start interactively annotating parts of the screen to screenshot.
```

File: pages/linux/flameshot.md [hunk 3]
```diff
@@ -2,24 +2,32 @@
...
 `flameshot gui`
 
-- Take a full screen screenshot:
+- Take a full screenshot (all monitors):
 
 `flameshot full`
 
+- Take screenshot from monitor 1:
+
+`flameshot screen --number 1`
```
Comment (CleanMachine1) [inline pages/linux/flameshot.md:21]:
```suggestion
`flameshot screen --number {{1}}`
```

Our syntax for variables

File: pages/linux/flameshot.md [hunk 4]
```diff
@@ -2,24 +2,32 @@
...
+
 - Set the save path to write screenshots to:
 
 `flameshot full --path {{path/to/directory}}`
 
 - Delay the screenshot for N milliseconds and output to clipboard:
 
 `flameshot full --delay {{2000}} --clipboard`
+
+- Export the screenshot to standard-output:
```
Comment (CleanMachine1) [inline pages/linux/flameshot.md:31]:
```suggestion
- Take a screenshot and export it to standard-output:
```

**Review by CleanMachine1 — APPROVED** [review]
LGTM after these!

**Comment by mmahmoudian** [comment]
> I will give some of my suggestions now

I appreciate the suggestions. The explanations made total sense. Thanks.

> however before merging can happen, you need to sign the CLA.

Done.

> ps, I really like flameshot, ever since starting with Linux, I found it and its the much better replacement for snipping tool.

I'm glad to hear that. Please feel free to give us feedback to make it even better 👍🏼

File: pages/linux/flameshot.md [hunk 5]
```diff
@@ -2,24 +2,32 @@
...
-> More information: <https://flameshot.js.org>.
+> More information: <https://flameshot.org>.
 
-- Launch flameshot in GUI mode:
+- Launch flameshot with a simpler interactive mode:
 
 `flameshot launcher`
 
-- Take a screenshot by clicking and dragging:
+- Launch flameshot and immediately start interactively annotating parts of the screen to screenshot.
```
Comment (navarroaxel) [inline pages/linux/flameshot.md:11]:
```suggestion
- Launch flameshot and immediately start interactively annotating parts of the screen to screenshot:
```

File: pages/linux/flameshot.md [hunk 6]
```diff
@@ -2,24 +2,32 @@
...
 
 `flameshot gui`
 
-- Take a full screen screenshot:
+- Take a full screenshot (all monitors):
 
 `flameshot full`
 
+- Take screenshot from monitor 1:
```
Comment (sbrl) [inline pages/linux/flameshot.md:19]: Just the teensiest of suggestions
```suggestion
- Take a screenshot from monitor 1:
```

**Review by sbrl — APPROVED** [review]
Looks ok to  me after @navarroaxel's comment is addressed. I've also left a small one below for you too

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**